src/lib/y2partitioner/widgets/configure.rb

```diff
@@ -0,0 +1,280 @@
+# encoding: utf-8
+
+# Copyright (c) [2018] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "cwm"
+require "y2partitioner/widgets/reprobe"
+
+Yast.import "Stage"
+Yast.import "Popup"
+Yast.import "Arch"
+Yast.import "PackageCallbacks"
+Yast.import "PackageSystem"
+
+module Y2Partitioner
+  module Widgets
+    # "Configure" menu button used to run the corresponding YaST clients to
+    # activate several storage technologies
+    #
+    # Most of the behavior of this widget is a direct translation of the old
+    # Yast::PartitioningEpAllInclude from yast-storage
+    class Configure < CWM::CustomWidget
```

Comment:
Well, most of the times it returns a `MenuButton` and sometimes it returns an `Empty`. So I considered `ConfigureWidget` (since `ConfigureMenuButton` was not fully precise)... then I realized `Widgets::ConfigureWidget` was redundant... and I ended up with no suffix. :wink:

So typical twisted developer mind process.